### DISCUSSION TITLE
Equivalent of jest's identity-object-proxy for CSS modules

### DISCUSSION THREAD

**Discussion by ericchernuka** [vitest-dev/vitest#1180] [Q&A]
I'm just starting to convert our Jest projects, and I was wondering if someone knew of a way to handle writing CSS modules names as strings like (identity-obj-proxy)[https://github.com/keyz/identity-obj-proxy] did in Jest.

**Comment by Bohooslav** [comment]
You actually can do something like this.

```
  resolve: {
    alias: [
      {
        find: /.*\.(?:scss|sass|css)$/,
        replacement: fileURLToPath(new URL('./node_modules/identity-obj-proxy/src/index.js', import.meta.url)),
      },
      ...
```

**Reply by sheremet-va** [reply]
Actually, the proxy-like behaviors for CSS modules is built-in in Vitest: https://vitest.dev/config/#css-modules-classnamestrategy

**Reply by Bohooslav** [reply]
Alas, any of the options from docs didn't work out for me.
